osu.Game/Screens/OnlinePlay/Lounge/DrawableLoungeRoom.cs

```diff
@@ -191,6 +191,9 @@ public partial class PasswordEntryPopover : OsuPopover
 
             protected override bool BlockNonPositionalInput => true;
 
+            // When a room is clicked, it already plays a click sound, which clashes pretty badly with the pop in sound.
```

Comment:
I'm not a huge fan of these local overrides. But if they are to exist, they need an attached comment to explain why they exist. I've added one here.

In this case, I think I'd sooner from the click sound from the panel before than pop in sound. It's very weird having things appear in some places with an attached sound but not others..

Reply:
Can you try this? I think it sounds better:

```diff
diff --git a/osu.Game/Screens/OnlinePlay/Lounge/DrawableLoungeRoom.cs b/osu.Game/Screens/OnlinePlay/Lounge/DrawableLoungeRoom.cs
index a45583a2ec..5cf2f91ff4 100644
--- a/osu.Game/Screens/OnlinePlay/Lounge/DrawableLoungeRoom.cs
+++ b/osu.Game/Screens/OnlinePlay/Lounge/DrawableLoungeRoom.cs
@@ -170,7 +170,6 @@ protected override bool OnClick(ClickEvent e)
 
             if (Room.HasPassword.Value)
             {
-                sampleJoin?.Play();
                 this.ShowPopover();
                 return true;
             }
@@ -191,10 +190,6 @@ public partial class PasswordEntryPopover : OsuPopover
 
             protected override bool BlockNonPositionalInput => true;
 
-            // When a room is clicked, it already plays a click sound, which clashes pretty badly with the pop in sound.
-            // Dunno about this one. I'd probably remove the click sound from the panel in cases they are password protected and play these pop in / out sounds.
-            protected override bool PlayPopInOutSamples => false;
-
             public PasswordEntryPopover(Room room)
             {
                 this.room = room;

```